### ISSUE TITLE
We need separate the test codes and example codes.

### ISSUE THREAD

**Issue by xpqiu** [fastnlp/fastNLP#67]
**Is your feature request related to a problem? Please describe.**  问题是什么

We need separate the test codes and example codes.

**Describe the solution you'd like** 解决方案是什么

Replace them into the different directories. 'test' directory is just for the unit/function test.

**Describe alternatives you've considered** 其他解决方案

**Additional context** 备注

**Comment by FengZiYjun** [comment]
Yes, I am going to create a directory named "scripts" in which there are only runable codes.
